iot_config/migrations/4_regions.sql

```diff
@@ -1,5 +1,5 @@
 create table regions (
-    region int primary key not null,
+    region text primary key not null,
```

Comment:
Does this require a new migration with an update table?

Reply:
nope; already reset the db. @madninja wants to continue to collapse these migrations until we're out of the "prototyping" phase